app/views/content/ways-to-train.md

```diff
@@ -265,4 +298,5 @@ Entry criteria may vary by teacher training provider. Ask them about eligibility
 A career change into teaching allows you to use your experience and
 passion to inspire young people.
 
-[Now Teach](https://nowteach.org.uk/) and [Transition to Teach](https://www.transitiontoteach.co.uk/) can support you with your career change.
+As well as the options above, [Now Teach](https://nowteach.org.uk/) and [Transition to Teach](https://www.transitiontoteach.co.uk/) are services that can specifically help those changing careers.
```

Comment:
"those" feels slightly awkward - are we deliberately not saying "you" here?

(If that's a deliberate choice so as not to promote these services quite as much, that's fine)

Reply:
have gone in and changed this to 'you'